components/exercises/chapter-5-oscar/Everything/WeatherApp.tsx

```diff
@@ -0,0 +1,162 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import { every } from './every';
+import './styles.css';
+
+export type WeatherData = {
+  name?: string;
+  main?: {
+    temp: number;
+    temp_min: number;
+    temp_max: number;
+    feels_like: number;
+    humidity: number;
+    pressure: string;
+  };
+  weather?: [{ description: string }];
+  wind?: {
+    speed: number;
+  };
+};
+
+const API_KEY = '297cdcb5306dab20d8338f39e1459828';
```

Comment:
Can you add a comment like "This key is free and public" so there's no harm on leaving it right here

Reply:
added, thank you